model/jpa/src/main/java/org/keycloak/connections/jpa/updater/liquibase/custom/CustomCreateIndexChange.java

```diff
@@ -55,6 +56,8 @@
 public class CustomCreateIndexChange extends CreateIndexChange {
     private static final Logger logger = Logger.getLogger(CustomCreateIndexChange.class);
     private int indexCreationThreshold;
```

Comment:
It looks like this needs to be a long too.

Reply:
It seems that I was too lazy here, added in next commit.